demo/midgard-node/src/fibers/merge.ts

```diff
@@ -0,0 +1,58 @@
+import { DatabaseError } from "@/database/utils/common.js";
+import { Lucid, AlwaysSucceedsContract, Globals } from "@/services/index.js";
+import { StateQueueTx } from "@/transactions/index.js";
+import { TxSignError, TxSubmitError } from "@/transactions/utils.js";
+import * as SDK from "@al-ft/midgard-sdk";
+import { SqlClient } from "@effect/sql/SqlClient";
+import { Effect, pipe, Schedule } from "effect";
+
+export const mergeAction: Effect.Effect<
+  void,
+  | SDK.Utils.CmlDeserializationError
+  | SDK.Utils.DataCoercionError
+  | SDK.Utils.HashingError
+  | SDK.Utils.LinkedListError
+  | SDK.Utils.LucidError
+  | SDK.Utils.StateQueueError
+  | DatabaseError
+  | TxSubmitError
+  | TxSignError,
+  Lucid | AlwaysSucceedsContract | SqlClient | Globals
```

Comment:
Shouldn't this be `SqlClient.SqlClient` (preferably our own `Database`)?
```suggestion
  Lucid | AlwaysSucceedsContract | Database | Globals
```